src/sql/workbench/contrib/queryplan2/browser/utils.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export function removeLineBreaks(str: string): string {
```

Comment:
I'd suggest just putting this in https://github.com/microsoft/azuredatastudio/blob/ce612a3d9613a1857772150892e5f9abe1dd44f7/src/sql/base/common/strings.ts